README.md

```diff
@@ -5,6 +5,12 @@ PROCESS is the reactor systems code at the [UK Atomic Energy Authority](https://
 
 PROCESS was originally a Fortran code, but is currently a mixture of Python and Python-wrapped Fortran; the eventual aim is to have an entirely Python code base. In order to use PROCESS, the Fortran must be compiled and a Python-Fortran interface generated for the Python to import. Once built, it can be installed and run as a Python package.
 
+
+<figure markdown>
+![BlenderProcessImage](documentation/proc-pages/images/blender images/initial_collage.png){ width="100%"}
+<figcaption>Split view of DEMO-like reactor (generated using fusrr-pipeline).</figcaption>
+</figure>
```

Comment:
I could add the link to the fusrr repo when it is made public?

Reply:
Yeah, that's a good idea.